src/ophys_etl/modules/suite2p_registration/suite2p_utils.py

```diff
@@ -272,9 +276,16 @@ def optimize_motion_parameters(initial_frames: np.ndarray,
                     'smooth_sigma': -1,
                     'smooth_sigma_time': -1}
     logger('Starting search for best smoothing parameters...')
+    sub_frames = load_representitive_sub_frames(
+        suite2p_args['h5py'],
+        suite2p_args['h5py_key'],
+        trim_frames_start,
+        trim_frames_end,
+        n_batches=n_batches,
+        batch_size=suite2p_args['batch_size'])
     start_time = time()
-    for param_spatial, param_time in zip(params_spatial.flatten(),
-                                         params_time.flatten()):
+    for param_spatial, param_time in product(smooth_sigmas,
```

Comment:
you are redeclaring these variables from above. I don't think you meant to do that